File: Javascript/URI/01-Iniciantes/1000-1020/1007.js
let values = [4];
for (let i = 0; i < 4; i++)
{
    values[i] = parseInt(prompt(`Digite o valor ${i + 1}`));
}
console.log("DIFERENCA = " + (values[0] * values[1] - values[2] * values[3]));
document.write("DIFERENCA = " + (values[0] * values[1] - values[2] * values[3]));

File: Cplusplus/URI/01-iniciantes/01-Nivel-Concluido/1007.cpp
#include <iostream>

using namespace std;

int main()
{
    int aValue, bValue , cValue , dValue, result;
    cout << "Digite o valor de A" << endl;
    cin >> aValue;
    cout << "Digite o valor de B" << endl;
    cin >> bValue;
    cout << "Digite o valor de C" << endl;
    cin >> cValue;
    cout << "Digite o valor de D" << endl;
    cin >> dValue;
    result = aValue * bValue - cValue * dValue;
    cout << "DIFERENCA = "<< result << endl;
    return 0;
}